### PR TITLE
docs(Knobs): fix registerKnob()

### PR DESCRIPTION
**Pull request by layershifter**
## Problem

There was a bug in implementation of `registerKnob()` and usage of `setKnobs()`, we used unactual value from state to check that knobs with same name will not be registered.

It caused an issue that you can't to use the same name for knob in Children and Shorthand examples, for example: https://github.com/stardust-ui/react/pull/1489/files#diff-50c810d56072655fb0b3d6c8703d58a3L7

### CODE REVIEW

File: docs/src/components/ComponentPlayground.tsx [hunk 1]
```diff
@@ -12,6 +12,17 @@ type ComponentPlaygroundProps = {
   componentName: string
 }
 
+const NoopKnobProvider: React.FunctionComponent = props => {
+  const knobContext = React.useContext(unstable_KnobContext)
+  const noopContext = { ...knobContext, registerKnob: _.noop, unregisterKnob: _.noop }
+
+  return (
+    <unstable_KnobContext.Provider value={noopContext}>
```
Comment (layershifter) [inline docs/src/components/ComponentPlayground.tsx:20]: I don't have a better idea now, but I definitely want to remove this from knobs:

image

File: docs/src/components/ComponentDoc/ComponentExample/ComponentExampleKnobs.tsx [hunk 2]
```diff
@@ -23,6 +23,7 @@ const ComponentExampleKnobs = createComponent({
           {({ classes }) => (
             <CodeSnippet
               className={classes}
+              copyable={false}
```
Comment (layershifter) [inline docs/src/components/ComponentDoc/ComponentExample/ComponentExampleKnobs.tsx:26]: With #1477 was introduced an ability to copy contents of snippet, but there are no sence to copy values from knobs

File: docs/src/components/ComponentPlayground.tsx (hunk 1 above)
Comment (mnajdova) [reply docs/src/components/ComponentPlayground.tsx:20]: The Tooltip component was merged, please consider those changes as well.

**Review by bmdalex — APPROVED** [review]

File: docs/src/components/ComponentPlayground.tsx (hunk 1 above)
Comment (layershifter) [reply docs/src/components/ComponentPlayground.tsx:20]: Done 👍

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**